src/Adapter/Product/Repository/ProductMultiShopRepository.php

```diff
@@ -29,13 +29,18 @@
 namespace PrestaShop\PrestaShop\Adapter\Product\Repository;
 
 use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Driver\Exception;
+use Doctrine\DBAL\Exception as ExceptionAlias;
 use Doctrine\DBAL\FetchMode;
+use Doctrine\DBAL\Query\QueryBuilder;
 use ObjectModel;
 use PrestaShop\PrestaShop\Adapter\Category\Repository\CategoryRepository;
```

Comment:
Doctrine + ObjectModel , on a same `AbstractMultiShopObjectModelRepository`:thinking:

There's a design problem here, doctrine usages should live inside the Prestashop bundle, not inside the adapter namespace.

This repository is already too big, we need to split here.

Reply:
+ This repository does not have any interface.